slave/buildslave/commands/base.py

```diff
@@ -147,7 +155,7 @@ def __init__(self, builder, stepId, args):
         self.args = args
         self.startTime = None
 
-        missingArgs = filter(lambda arg: arg not in args, self.requiredArgs)
```

Comment:
this is also 3.3+ compatible but not according to PEP8 standards. That's why i have replaced the lambdas with a def function.